core/file_and_image.md

```diff
@@ -0,0 +1,3 @@
+This section of the Media Guide needs to be written.
+
+See our [Contributing](contributing.md) section of the Guide to find out how to get involved and help out.
```

Comment:
I think that this link won't work since we're one level up. Applies to few more pages below.